src/rules.c

```diff
@@ -550,7 +550,7 @@ void SchemaPrefixes_RemoveSpec(StrongRef ref) {
         array_del_fast(node->index_specs, j);
         if (array_len(node->index_specs) == 0) {
           // if all specs were deleted, remove the node
-          TrieMap_Delete(SchemaPrefixes_g, prefixes[i], strlen(prefixes[i]), (freeCB)SchemaPrefixNode_Free);
+          TrieMap_Delete(SchemaPrefixes_g, prefixes[i], sdslen(prefixes[i]), (freeCB)SchemaPrefixNode_Free);
```

Comment:
If the triemap doesn't own its data, and is referring to external, unowned data, then I understand why `prefixes` must point to `mut` data. But otherwise, both it, and the `spec` should be pointers to `const`. As it happens, `TrieMap_Delete` is does accept `const str`. So I highly doubt that it doesn't own its own data.